src/omero_cli_render.py

```diff
@@ -514,29 +514,30 @@ def set(self, args):
         rangelist = []
         colourlist = []
         for (i, c) in newchannels.iteritems():
+            i += 1
```

Comment:
I am still unclear why this offset necessary. Looking at https://github.com/IDR/idr0047-neuert-yeastmrna/pull/11#discussion_r248256551, could the confusion come from the assumption that the channels loaded from the YAML/JASON files are 0-indexed?

Reply:
Ah, so that's why the export via 'info' command import via 'set' didn't work. Ok, maybe it's better in that case to not increase by one, but fix the output of the 'info' command.